### PR TITLE
Update README to address issues

### PR DESCRIPTION
**Pull request by ohohoreilly**
Update README to address issues provided in https://github.com/GoogleCloudPlatform/kubernetes/issues/7122 by @lavalamp
Providing a bit more clarity to how to test and develop the kubernetes Web UI.

### CODE REVIEW

File: www/README.md [hunk 1]
```diff
@@ -87,7 +91,7 @@ angular.module('kubernetesApp.config', [])
 See [master/components/README.md](master/components/README.md).
 
 ### Testing
-Currently kuberntes-ui includes both unit-testing (run via [Karma](http://karma-runner.github.io/0.12/index.html)) and
+Currently kuberntes/www includes both unit-testing (run via [Karma](http://karma-runner.github.io/0.12/index.html)) and
```
Comment (brendandburns) [inline www/README.md]: s/kuberntes/kubernetes/

**Comment by brendandburns** [comment]
I'm happy to merge this, although I don't believe that it addresses all of @lavalamp 's comments about the README.  If this is understood to be a partial addressing, and that there will be subsequent additional PRs, I'm happy to merge this.  (assuming the one nit is addressed ;)

**Comment by preillyme** [comment]
@brendanburns yeah for sure it doesn't cover all the concerns at all. I just wanted to address some of the gaps ASAP as I work on a better solution is all.

File: www/README.md (hunk 1 above)
Comment (preillyme) [inline www/README.md]: I've fixed the typo.

**Comment by brendandburns** [comment]
sg, merged.

**Pull request merged**